docs/config.md

```diff
@@ -0,0 +1,104 @@
+# Configuring the HYPR Authenticator
+To use the HYPR authenticator with WSO2 Identity Server, first you need to configure  the authenticator with
+WSO2 Identity Server. The following steps provide instructions on how to configure the HYPR authenticator with
+WSO2 Identity Server using a sample app.
+
+In order to test the approach, First an end user account should be registered in the Application created via the 
+HYPR Control Center and the end user should have the HYPR mobile application installed in the smartphone, as well as 
+the smartphone registered with the end user account.
+
+After deploying the HYPR Authenticator to WSO2 IS, the Authenticator can be configured from the
+WSO2 IS Console.
+
+## Requirements
+To use the connector, you'll need:
+
+- A configured HYPR environment.
+- A HYPR user account and mobile app to use for testing.
+
+Note: Get the support from HYPR to configure a HYPR Application via the HYPR Control Center.
+
+## Setup and Installing HYPR Connector
+
+**Step 1:** Extract the project artifacts
+- Clone the `identity-outbound-auth-hypr` repository.
+- Build the project by running ```mvn clean install``` in the root directory.
+
+Note : The latest project artifacts can also be downloaded from the Connector Store.
+
+**Step 2:** Deploying the HYPR Authenticator
+
+- Navigate to `identity-outbound-auth-hypr/components` → `org.wso2.carbon.identity.application.authenticator.hypr` 
+→ `target`.
+- Copy the `org.wso2.carbon.identity.application.authenticator.hypr-1.0.0-SNAPSHOT.jar` file.
+- Navigate to `<IS_HOME>/repository/components/dropins`.
+- Paste the `.jar` file into the dropins directory.
+- Alternatively it's possible to drag and drop the `.jar` file to the dropins directory.
+- Next navigate to `identity-outbound-auth-hypr/components` → 
+`org.wso2.carbon.identity.application.authenticator.hypr.common` → `target`.
+- Copy the `org.wso2.carbon.identity.application.authenticator.hypr.common-1.0.0-SNAPSHOT.jar` file.
+- Navigate to `<IS_HOME>/repository/components/lib` directory and paste the `.jar` file.
+
+**Step 3:** Deploying the HYPR REST API
+- Navigate to `identity-outbound-auth-hypr/components` → `org.wso2.carbon.identity.application.authenticator.hypr.rest` 
```

Comment:
Numbering for procedures

Reply:
Resolves via [3ef7a2c](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/2/commits/3ef7a2caa1aafeb0302cc6662760cbe471bd299d).